hedera-mirror-rest/api/v1/openapi.yml

```
@@ -1423,6 +1475,13 @@ components:
           description: The network's total supply of hbars in tinybars
           example: "5000000000000000000"
           type: string
+    NftAllowancesResponse:
+      type: object
+      properties:
+        allowances:
+          $ref: "#/components/schemas/NftAllowances"
+        links:
+          $ref: "#/components/schemas/NftLink"
```

We usually just link to the common one. You can delete NftLink. This will allow a shared generated model.

```suggestion
          $ref: "#/components/schemas/Links"
```